app/Models/Farmasi/ResepObat.php

```diff
@@ -65,9 +65,12 @@ public function scopeJenisKunjungan(Builder $query, string $jenisKunjungan = 'se
                             ->where('reg_periksa.kd_poli', '!=', 'IGDK');
                     case 'ranap':
                         return $query->where('resep_obat.status', 'Ranap');
-                    case 'igd':
+                    case 'igd_ralan':
                         return $query->where('resep_obat.status', 'Ralan')
                             ->where('reg_periksa.kd_poli', '=', 'IGDK');
+                    case 'igd_ranap':
+                        return $query->where('resep_obat.status', 'Ranap')
```

Comment:
Querynya udah benar sebelumnya, nggak perlu pisah status ralan ranap IGD, langsung digabung aja buat ranapnya.
```suggestion
                        return $query->whereIn('resep_obat.status', ['Ralan', 'Ranap'])
```